Jenkinsfile

```diff
@@ -24,6 +32,7 @@ pipeline
             steps
             {
                 sh """
+                    apk add --no-cache --repository http://dl-cdn.alpinelinux.org/alpine/v3.4/main/ maven;
```

Comment:
The installation of Maven (mvn) should be added to image jenkins-agent-common
Consult @aleksandrOranskiy